src/app/shared/form/fancy-duration.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Input, DoCheck } from '@angular/core';
+import { BaseModel } from '../model/base.model';
+
+@Component({
+  selector: 'publish-fancy-duration',
+  styleUrls: ['fancy-field.component.css', 'fancy-duration.component.css'],
+  template: `
+    <div class="field small inline" [class.tiny]="tiny">
+      <h4 *ngIf="label">
+        <label [attr.for]="hoursName">{{label}}</label>
+      </h4>
+      <template [ngIf]="!model">
+        <input [id]="hoursName" type="number" disabled=true/>
+        <b>:</b>
+        <input [id]="minutesName" type="number" disabled=true/>
+        <b>:</b>
+        <input [id]="secondsName" type="number" disabled=true/>
+      </template>
+      <template [ngIf]="model">
+        <input [id]="hoursName" type="number" min="0" [ngModel]="hours | padzero"
+          (ngModelChange)="set('hours', $event)" [class.changed]="hoursChanged"/>
+        <b>:</b>
```

Comment:
I guess my reasoning was partially non-semantic, since I needed to style the `:` without changing the `<input>` font style as well, so it needed its own element.  Which also let me set specific margins on that single character, to space the inputs properly.

But if I pretend I was thinking semantically for a second... the `<b>` tag "represents a span of text stylistically different from normal text, without conveying any special importance or relevance".  Which is exactly what I'm after - stylistically different but not important.